packages/babel-plugin-transform-react-inline-elements/README.md

```diff
@@ -1,11 +1,58 @@
 # babel-plugin-transform-react-inline-elements
 
-Turn JSX elements into exploded React objects
+Replaces the `React.createElement` function with one that is more optimized for production: `babelHelpers.jsx`.
+
+<blockquote class="babel-callout babel-callout-info">
```

Comment:
Awesome stuff!

I think the only thing is to keep this <blockquote> in the website readme and then import the rest of the readme from here.